src/server/scripts/Northrend/Naxxramas/boss_faerlina.cpp

```diff
@@ -71,23 +74,28 @@ class boss_faerlina : public CreatureScript
             me->SummonCreature(NPC_NAXXRAMAS_WORSHIPPER, 3344.3f, -3618.31f, 261.08f, 4.69494f);
             me->SummonCreature(NPC_NAXXRAMAS_WORSHIPPER, 3356.71f, -3620.05f, 261.08f, 4.57276f);
             me->SummonCreature(NPC_NAXXRAMAS_WORSHIPPER, 3350.26f, -3619.11f, 261.08f, 4.67748f);
-
-            // Followers
             if (Is25ManRaid())
             {
                 me->SummonCreature(NPC_NAXXRAMAS_FOLLOWER, 3347.49f, -3617.59f, 261.0f, 4.49f);
                 me->SummonCreature(NPC_NAXXRAMAS_FOLLOWER, 3359.64f, -3619.16f, 261.0f, 4.56f);
             }
         }
 
-        void JustSummoned(Creature* cr) override { summons.Summon(cr); }
+        void JustSummoned(Creature* cr) override
+        {
+            summons.Summon(cr);
+        }
 
         void Reset() override
         {
             BossAI::Reset();
             events.Reset();
             summons.DespawnAll();
             SummonHelpers();
+            if (GameObject* go = me->FindNearestGameObject(SPIDER_WEB_ENTRANCE, 100.0f))
```

Comment:
@Si1ker not necessary, just a tip: can be shorter using `auto go` instead of having to specify the type